portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase

```diff
@@ -30,12 +30,13 @@ definition {
 	}
 
 	@description = "Automation ID: LRAC-8108 | Test Summary: Add date source page that allows user to choose type of data source"
-	@ignore = "true"
 	@priority = "5"
 	test AddDataSourcePageAllowsUserToChooseWhatKind {
+		ACUtils.launchAC();
 
-		// TODO LRAC-8108 AddDataSourcePageAllowsUserToChooseWhatKind
+		ACNavigation.goToSettings();
 
+		ACSettings.assertDataSource(dataSources = "Liferay DXP,Salesforce,CSV File");
```

Comment:
Hi @AfonsoHSantos, I think the test would not only be assert, I believe it would be to verify if the user can create these 3 types of data sources. What do you think @alexcandidos ?

Reply:
Yeah, I think we should see that we are able to at least click on those 3 options and assert the option related to that type of Data Source... for example, validating that by selecting DXP, a token option will be available, choosing a CSV something created to it will be shown, and the same for Salesforce.